s4e-web/src/app/views/map-view/map/map.component.ts

```diff
@@ -26,37 +24,37 @@ import ImageWrapper from 'ol/Image';
 })
 export class MapComponent implements OnInit, OnDestroy {
   static readonly DEFAULT_ZOOM_LEVEL = 10;
-
-  get overlays(): UIOverlay[] {
-    return this._overlays;
-  }
-
-  @Input() set overlays(value: UIOverlay[]) {
-    this._overlays = value;
-    this.overlays$.next(value);
-  }
-
-  private _overlays: UIOverlay[] = [];
-
-  private overlays$: BehaviorSubject<UIOverlay[]> = new BehaviorSubject([]);
   @Output() viewChanged = new EventEmitter<ViewPosition>();
-  private baseLayer: Layer;
-  private map: Map;
-  private activeScene$ = new ReplaySubject<Scene | null>(1);
   @ViewChild('linkDownload', {read: ElementRef}) linkDownload: ElementRef;
   @Input() isWorking: boolean = false;
   @Output() working = new EventEmitter<boolean>();
   activeView$ = new BehaviorSubject<ViewPosition>({
     centerCoordinates: this.CONFIG.projection.coordinates,
     zoomLevel: MapComponent.DEFAULT_ZOOM_LEVEL
   });
+  private overlays$: BehaviorSubject<UIOverlay[]> = new BehaviorSubject([]);
+  private baseLayer: Layer;
+  private map: Map;
+  private activeScene$ = new ReplaySubject<Scene | null>(1);
 
   constructor(
     private CONFIG: S4eConfig,
     private _loaderService: NgxUiLoaderService,
     private _notificationService: NotificationService,
     private _sessionQuery: SessionQuery
-  ) {}
+  ) {
+  }
+
+  private _overlays: UIOverlay[] = [];
+
+  get overlays(): UIOverlay[] {
```

Comment:
I've heard a lot of bad opinions about using `getters` and `setters` in typescript (better option would be `getOverlays()` function). They can provide hard to find bugs and inconsistent behaviors. Is there any reason why you're using it?